### DISCUSSION TITLE
I am facing the access issue for NPM while running the pipeline in docker

### DISCUSSION THREAD

**Discussion by SaiTharun546** [directus/directus#17178] [Archive]
I am unable to run the pipeline because the NPM is trying to create the directory in docker container
And showing the error like below:
Error:
> engineering_data@1.0.0 start
> npx directus bootstrap && npx directus start
npm ERR! code EACCES
npm ERR! syscall open
npm ERR! path /.npm/_cacache/tmp/63232332
npm ERR! errno -13
npm ERR! 
npm ERR! Your cache folder contains root-owned files, due to a bug in
npm ERR! previous versions of npm which has since been addressed.
npm ERR! 
npm ERR! To permanently fix this problem, please run:
npm ERR!   sudo chown -R 6005:6005 "/.npm"
npm ERR! Log files were not written due to an error writing to the directory: /.npm/_logs

**Comment by paescuj** [comment]
Apparently, discussed on https://discord.com/channels/725371605378924594/1064469579465760788

**Reply by SaiTharun546** [reply]
Currently we are using directus latest image
